python/vtool/data.py

```diff
@@ -678,11 +694,8 @@ def export_data(self, comment, selection=None):
             selection = []
         library = self._initialize_library()
 
-        controls = []
         if selection:
-            for thing in selection:
-                maya_lib.core.has_shape_of_type(thing, 'nurbsCurve')
```

Comment:
Was this lookup intentional or intended to be part of an if statement originally? @louisVottero